lib/tasks/import.rake

```diff
@@ -2,6 +2,8 @@ namespace :import do
   desc 'Import an organisation\'s details and content items'
   task :organisation, [:slug] => :environment do |_, args|
     raise 'Missing slug parameter' unless args.slug
+    Rails.logger = Logger.new(STDOUT)
```

Comment:
This need to be wrapped in a:
```ruby
unless Rails.env.test?
...
end
```
as I'm getting log messages to the STDOUT when I run `rspec`.